### ISSUE TITLE
Bug: Subscribe email form submission is not styled  

### ISSUE THREAD

**Issue by bdougie** [open-sauced/landing-page#78]
### Describe the bug

<img width="704" alt="Screen Shot 2022-12-10 at 2 44 29 PM" src="https://user-images.githubusercontent.com/5713670/206878136-72e99b73-b9b1-4afc-b4cc-a745d2e54378.png">

This should be styled like below

<img width="709" alt="Screen Shot 2022-12-10 at 2 43 37 PM" src="https://user-images.githubusercontent.com/5713670/206878101-112bf261-e7d6-44dd-a636-9efbf1d3f439.png">

### Steps to reproduce

Scroll to the bottom of the page and type in an email. Then click out the form. 

### Affected services

opensauced.pizza

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Contributing Docs

- [ ] I agree to follow this project's Contribution Docs

**Comment by stefanvucic** [comment]
Hello, @davidnagli as I can see this issue is resolved?

**Comment by Tickerabucky2** [comment]
Hello, @davidnagli as I can see this issue is resolved?

**Comment by davidnagli** [comment]
@Tickerabucky2 I think you tagged me in the wrong issue. But this issue seems to be about how autofill removes the styling from the input. I think this should be an easy fix.

**Comment by Tickerabucky2** [comment]
Sorry, @davidnagli I have made a mistake, I copied the message from the previous issue.

@bdougie This issue appears only after the user selects the proposed auto-complete option. Typing something inside the input element and clicking outside the form will not affect the element styling. This issue is presented after the [PR ](https://github.com/open-sauced/landing-page/pull/76) is merged. 

If it is fine for you, disabling the autocomplete option will solve this issue?
